addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -76,6 +76,7 @@ const Wysiwyg = Widget.extend({
         this.$editable.data('oe-id', options.recordInfo.res_id);
         document.addEventListener('mousedown', this._onDocumentMousedown, true);
         this.$editable.on('blur', this._onBlur);
+        $('.o_not_editable').attr('contentEditable', false);
```

Comment:
Don't do this. See discussion at https://github.com/odoo/odoo/pull/82222#discussion_r818819590 with the solution being currently developed at https://github.com/odoo/odoo/pull/87347

Reply:
We know, that's from here that the discussion started :wink: This task is currently waiting for that fix to be merged indeed :wink:

Reply:
Oh alright, sorry then. It's getting quite hard to follow the maze of different PRs on this subject... ^^'